serial_port: "/dev/ttyXYZ0"
serial_baudrate: 115200
serial_timeout: 1000
websocket_port: 8080

